trunk/3rdparty/srs-bench/blackbox/hevc_test.go

```diff
@@ -995,11 +994,14 @@ func TestSlow_SrtPublish_HlsPlay_HEVC_Basic(t *testing.T) {
 		defer wg.Done()
 		<-svr.ReadyCtx().Done()
 
+		// wait for ffmpeg
+		time.Sleep(3 * time.Second)
+
```

Comment:
No need to sleep here, `ffmpeg` can publish stream immediately after `srs` ready.